buildkite/src/Jobs/Test/Nightly.dhall

```diff
@@ -0,0 +1,33 @@
+let S = ../../Lib/SelectFiles.dhall
+
+let JobSpec = ../../Pipeline/JobSpec.dhall
+let Pipeline = ../../Pipeline/Dsl.dhall
+
+let DeployTestnet = ../../Command/DeployTestnet.dhall
+
+
+let spec = DeployTestnet.DeploySpec::{
+  testnetLabel = "nightly",
+  workspace = "nightly",
+  deployCondition = "build.branch == 'compatible' && build.env('NIGHTLY') == 'true'",
+  preDeploy = "terraform destroy -auto-approve && cp /tmp/gcp_creds.json ../../../gcloud-keyfile.json",
```

Comment:
maybe add a post deploy operation for removing (or archiving) existing network block data?